src/radio_instance.c

```diff
@@ -462,9 +468,10 @@ static
 char*
 radio_instance_make_key(
     const char* dev,
-    const char* name)
+    const char* name,
+    RADIO_INTERFACE version)
 {
-    return g_strconcat(dev, ":", name, NULL);
+    return g_strdup_printf("%s:%s:%d", dev,name,version);
```

Comment:
Nitpick, no spaces between the parameters.

Reply:
Fixed, thanks!